[2024-10-16T01:10:05.210+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2024-10-16T01:05:00+00:00 [queued]>
[2024-10-16T01:10:05.214+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2024-10-16T01:05:00+00:00 [queued]>
[2024-10-16T01:10:05.214+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T01:10:05.214+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-10-16T01:10:05.214+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T01:10:05.219+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2024-10-16 01:05:00+00:00
[2024-10-16T01:10:05.222+0000] {standard_task_runner.py:55} INFO - Started process 815 to run task
[2024-10-16T01:10:05.223+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2024-10-16T01:05:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpvy3y2l6r']
[2024-10-16T01:10:05.225+0000] {standard_task_runner.py:83} INFO - Job 102: Subtask store_user
[2024-10-16T01:10:05.252+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user scheduled__2024-10-16T01:05:00+00:00 [running]> on host dcdd4c71b617
[2024-10-16T01:10:05.277+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2024-10-16T01:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-16T01:05:00+00:00
[2024-10-16T01:10:05.277+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2024-10-16T01:10:05.280+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-10-16T01:10:05.282+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-10-16T01:10:05.285+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20241016T010500, start_date=20241016T011005, end_date=20241016T011005
[2024-10-16T01:10:05.323+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-10-16T01:10:05.331+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
